README.md

```diff
@@ -17,3 +17,20 @@ and so on.
 For more information, see the Contiki website:
 
 [http://contiki-os.org](http://contiki-os.org)
+
+=====
+
+## 6LoWPAN Clicker Build Instructions
+
+**If you would like to build the IoT Creator Kit projects, go to https://github.com/CreatorKit/build and follow the instructions there.**
```

Comment:
contiki build instructions and build makefile has different command line arguments. so let's not confuse people by pointing to CreatorKit/build from here.